fix: add content faq (#335)
design-tokens/community-group: www/src/pages/faq.md

layout: ../layouts/Markdown.astro
---

## Is this ready to use?

> The Design Tokens Community Group (DTCG) specification is still in draft status as of October 2025, but it's closer than ever to reaching v1.0.

Reviewer: We are going to use a calendar date as the first version. This probably doesn’t make much sense, so maybe add “first stable version of _____” or whatever (not particular on wording)

```suggestion
The Design Tokens Community Group (DTCG) specification is still in draft status as of October 2025, but it's closer than ever to reaching 2025.10.
```

Production readiness:

- ✅ **Safe to use:** Many organizations are already implementing it
- ⚠️ **Still evolving:** The spec is in active development with ongoing refinements
- 📝 **Draft status:** Not yet a W3C Recommendation or final standard

What does this mean for you?

- ✅ The core format is stable enough for production use
- ✅ Major design systems and tools are already adopting it
- ⚠️ Expect potential changes before v1.0 finalization
- ⚠️ Some advanced features are still being refined
- 📋 Plan for possible migration adjustments when v1.0 releases

You can start using the specification for production in most cases, especially for new projects. However, make sure to stay informed of the latest updates:

> - Follow the [latest draft](https://www.designtokens.org/tr/drafts/) closely

Author: Linked to blog!

- [Superposition](https://www.superposition.design)
- [Supernova](https://www.supernova.ion)
- [Terrazzo](https://www.terrazzo.app)
- [Toolabs](https://www.toolabs.com)
- [TokensBrücke](https://www.figma.com/community/plugin/1254538877056388290)
- [Tokens Studio](https://www.tokens.studio)
- [Universal Design Tokens](https://www.github.com/universal-design-tokens/udt)
- [Zeplin](https://www.zeplin.io)
- [zeroheight](https://www.zeroheight.com)

## How does this relate to other token approaches?

The Design Tokens Community Group specification relates to other token approaches as a standardization layer that unifies previously fragmented, tool-specific formats. It provides:

- **Canonical syntax** replacing incompatible proprietary formats
- **Interoperability foundation** enabling cross-tool workflows
- **Validation framework** ensuring format consistency
- **Extension mechanism** preserving tool-specific capabilities

The specification **complements rather than replaces** token methodologies and architectural patterns developed by production design systems. It defines the format for token exchange while leaving organizational strategy to design system teams.

> As of 2025, the specification approaches v1.0.0 release with growing adoption across major design tools and transformation pipelines. Teams should evaluate DTCG migration based on interoperability requirements, tooling support, and long-term maintenance considerations.

Reviewer: ```suggestion
As of 2025, the specification approaches 2025.10 release with growing adoption across major design tools and transformation pipelines. Teams should evaluate DTCG migration based on interoperability requirements, tooling support, and long-term maintenance considerations.
```

## What is the relationship between JSON Schema and DTCG?

> JSON Schema is a declarative specification, a passive validator, and documentation format. It can be used to enforce documents to adhere to a strict format. It is a general purpose data interchange format.

Reviewer: I think this still has some remnants of confusing JSON and JSON Schema. JSON Schema is not a general purpose data interchange format. It’s not quite accurate to say it’s a validator, either; technically those are JSON Schema tools. 

```suggestion
[JSON Schema](https://json-schema.org/) is a tool for describing JSON structures.
```

DTCG is a specification to provide indivisible pieces of a design system such as colors, spacing, typography scale. It is purpose-built for design token exchange. It is both a way to declare your design system token schema and a way to deliver the actual content.

Similarities:

- Both use JSON syntax (objects, strings, numbers, arrays, booleans, null)
- Both support nested hierarchical structures
- Both are human-readable and machine-parsable
- Both use key-value pairs for data organization

> - Both can be validated against schemas

Reviewer: This is not accurate; JSON Schema is a way of _describing_ schemas, it itself can’t be validated against a schema

```suggestion
```

> While they may have some similarities, they have different purposes and features. DTCG has required properties, reserved keywords, type system and validation, standardized alias and reference system, type inheritance through groups, composite token structures, tooling and transformation expectations.

Reviewer: This paragraph doesn’t make sense to me, because I don’t understand what “required properties” and “reserved keywords” mean, especially in contrast to JSON Schema. We could mirror the previous paragraph, like so:

```suggestion
Differences:

- DTCG serves as both a way to declare your schema _and_ exchange token data in the same file (JSON Schema is not for data exchange)
- JSON Schema is a generic tool; DTCG is tailored for design systems and design tokens
- DTCG types map to specific concepts in design tools and web and mobile app development
```

🤔 something along those lines

layout: ../layouts/Markdown.astro
---

## Is this ready to use?

The Design Tokens Community Group (DTCG) specification is still in draft status as of October 2025, but it's closer than ever to reaching the first stable version, 2025.10.

Production readiness:

- ✅ **Safe to use:** Many organizations are already implementing it
- ⚠️ **Still evolving:** The spec is in active development with ongoing refinements

> - 📝 **Draft status:** Not yet a W3C Recommendation or final standard

Reviewer 2: ```suggestion
- 📝 **Spec status:** The DTCG spec is not on the W3C Standards Track
```

What does this mean for you?

- ✅ The core format is stable enough for production use
- ✅ Major design systems and tools are already adopting it
- ⚠️ Expect potential changes before v1.0 finalization
- ⚠️ Some advanced features are still being refined
- 📋 Plan for possible migration adjustments when v1.0 releases

You can start using the specification for production in most cases, especially for new projects. However, make sure to stay informed of the latest updates:

- Follow the [blog](/blog/) for official news and releases

> - Stay engaged with the [community](https://github.com/design-tokens/designtokens.org/issues) for ongoing discussions

Reviewer 2: ```suggestion
- Stay engaged with the [community](https://github.com/design-tokens/community-group/issues) for ongoing discussions
```

- Build with flexibility to accommodate spec changes
- Focus on the stable core features (basic token types, structure)

## What's the license?

The Design Tokens Community Group specification's [licensing](https://github.com/design-tokens/community-group/blob/content-faq/LICENSE.md) ensures the specification can be freely implemented by design tools, token transformation libraries, and design systems without licensing barriers.

- All Reports in this Repository are licensed by Contributors under the [W3C Software and Document License](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
- Contributions to Specifications are made under the [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/agreements/cla/).
- Contributions to Test Suites, Website and Content, and Other Software are made under the [W3C 3-clause BSD License](https://www.w3.org/Consortium/Legal/2008/03-bsd-license.html)

## How do I give feedback?

Join the [Design Tokens Community Group Github issues](https://github.com/design-tokens/community-group) (recommended):

- **New issues:** Open new issues for bugs, clarifications, or feature requests

> - **Pull requests:** Submit PRs for specific specification improvements or corrections

Author: Updated!
